staff/serena-mejias/cv/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <section>
+        <img src="#" alt="photo">
+        <h1>Serena Mejías Vázquez</h1>
+        <address>
+            Avda Europa, 4 bj 3ª
+            <br>
+            08110
+            <br>
+            <a href="serenamejvaz@gmail.com">serenamejvaz@gmail.com </a>
+        </address>
+    </section>
+    <section>
+        <h3>Expirience</h3>
+        <ul>
+            <li><b>2015 to January 2019</b></li>
```

Comment:
The W3C [discourages](https://developer.mozilla.org/es/docs/Web/HTML/Elemento/b#Notas) the use of `b` element.

Reply:
Also the use of ì`it's better `<em>`

Reply:
Done.